### PR TITLE
Dynaflow: Described the configuration and specific parameters

### PR DESCRIPTION
**Pull request by PerninGuillaume**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
N/A

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
This PR adds documentation for Dynaflow, a description of the specific simulation parameters that can be provided, and an example of a json configuration file.

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Comment by mathbagu** [comment]
This PR could be reworked if #157 is merged. In that proposal, I think some important information are missing, such as the way to install DynaFlow. There are mistakes (or inconsistencies) in the name of the implementation.

**Comment by gautierbureau** [comment]
> This PR could be reworked if #157 is merged. In that proposal, I think some important information are missing, such as the way to install DynaFlow. There are mistakes (or inconsistencies) in the name of the implementation.

Inconsistencies should be corrected. For the installation it is still a work in progress on dynaflow-launcher side so best to wait until it is done

File: pages/documentation/simulation/powerflow/dynaflow.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+The `VscAsGenerators` is an optional boolean property that defines if VSCs (Voltage Source Converters) are modeled as generators.
+The default value of this parameter is `true`.
+
+**LccAsLoads**  
+The `LccAsLoads` is an optional boolean property that defines if LCCs (Line Commutated Converters) are modeled as loads.
+The default value of this parameter is `true`.
+
+### Generic parameters
+Furthermore, DynaFlow only supports two of the generic parameters: 
+- NoGeneratorReactiveLimits
```
Comment (gautierbureau) [inline pages/documentation/simulation/powerflow/dynaflow.md:46]: noGeneratorReactiveLimits

File: pages/documentation/simulation/powerflow/dynaflow.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- PhaseShifterRegulationOn
```
Comment (gautierbureau) [inline pages/documentation/simulation/powerflow/dynaflow.md:47]: phaseShifterRegulationOn

File: pages/documentation/simulation/powerflow/dynaflow.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+You may have a description of these parameters [here](index.md#parameters). The other parameters are ignored.
+
+### Example
+
+This is an example of a load flow parameters file with specific parameters for DynaFlow:
+```json
+{
+  "version" : "1.0",
+  "voltageInitMode" : "PREVIOUS_VALUES",
```
Comment (gautierbureau) [inline pages/documentation/simulation/powerflow/dynaflow.md:57]: why having this parameter and others that are not supported ?

File: pages/documentation/simulation/powerflow/dynaflow.md (hunk 1 above)
Comment (PerninGuillaume) [reply pages/documentation/simulation/powerflow/dynaflow.md:46]: If you don't want to have capital for these parameters then I think the specifics parameters ought to have their first capital removed too

File: pages/documentation/simulation/powerflow/dynaflow.md (hunk 3 above)
Comment (PerninGuillaume) [reply pages/documentation/simulation/powerflow/dynaflow.md:57]: True enough, I will only keep the dynaflow-launcher supported parameters

File: pages/documentation/simulation/powerflow/dynaflow.md (hunk 1 above)
Comment (gautierbureau) [reply pages/documentation/simulation/powerflow/dynaflow.md:46]: I agree, every parameter should be written with no capital letter as it is the syntax used in the yaml and json

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**